henryPuma/assignments/treesBFS/levelOrderSuccessor/levelOrderSuccessor.js

```diff
@@ -0,0 +1,89 @@
+// Problem Statement #
+
+// Given a binary tree and a node, find the level order successor of the given node in the tree. The level order successor is the node that appears right after the given node in the level order traversal.
+
+//            (  1  )
+//           /       \
+//       (  2  )    (  3  )
+//       /     \       \
+// (  4  )  (  5  )   (  6  )
+
+// Given Node: 3
+// Level Order Successor: 4
+
+
+//             (  12  )
+//            /       \
+//       (  7  )    (  1  )
+//       /            /    \
+//  (  9  )       (  10  )(  5  )
+
+// Given Node: 9
+// Level Order Successor: 10
+
+class TreeNode {
+
+  constructor(val) {
+    this.value = val;
+    this.left = null;
+    this.right = null;
+  }
+};
+
+/*
```

Comment:
Nice work! Don't forget to include a note about the time and space complexity of your solution as well.